knowledge.md

```diff
@@ -172,8 +172,28 @@ chests:
 * **Silver**: Can have one of several effects
   * Upgrade your Aetherpool arm or armor
   * Chest explodes, dealing 70% max HP damage to any players and enemies nearby
-  * HoH only: chest may contain magicite
-  * EO only: chest may contain a demiclone tomestone
+  * HoH only: chest may contain magicite. There are four types of magicite:
+    * Inferno, Crag, and Vortex: Summons Ifrit, Titan, or Garuda respectively,
+      instantly killing all enemies on the floor (except bosses, to which they
+      deal a large amount of damage). Also grants a 10-second invulnerability
+      buff to all party members; this invulnerability takes effect immediately,
+      so magicites can be used as a panic button if you can't get out of an
+      AoE.
+    * Elder: Summons Odin, which has the same effect as the other magicites but
+      *also* one-shots bosses. Much rarer than the other types, and does not
+      appear at all before floor 31. **Note that elder magicite has been
+      reported not to work if you don't engage the boss in battle first!**
```

Comment:
Pretty sure this is just people being paranoid that it won't work, but doesn't hurt to mention just in-case

Reply:
I'm pretty sure that when I first heard about this, it was from a tweet that included video evidence, but I didn't think to save the link (and fat chance of finding it now, lol) so I left this as "has been reported". I do intend to test it out at some point - if I can ever find the time...